picard/util/__init__.py

```diff
@@ -609,3 +609,11 @@ def find_best_match(candidates, no_match):
     else:
         result = no_match
     return BestMatch(similarity=result.similarity, result=result, num_results=len(sorted_results))
+
+
+def callers(func, offset=3, length=1):
```

Comment:
Why an offset of 3 and length of 1? `offset=2` like in the call above seems to make sense. And when calling callers without specifying `length` my expectation would be to get all callers (or at least more than one).